controls/cis_rhel10.yml

```diff
@@ -880,7 +880,7 @@ controls:
       - l1_workstation
     status: automated
     notes: Review the availability of this package when the product is out.
-    rules:
+    related_rules:
```

Comment:
The comment in notes can be removed now.

Reply:
Done.